File: node/controllers/post-unit-to-army-roster-controller.ts
import { Request, Response } from 'express';
import { postUnitToArmyRosterToDb } from '../models/post-unit-to-army-roster-model';

export async function postUnitToArmyRosterController(req: Request, res: Response) {
    try {
        const unit_id = Number(req.body.unit_id) || 0;
        const unitData = await postUnitToArmyRosterToDb(unit_id);
        res.statusCode = 200;
        res.json(unitData);
    } catch (error) {
        res.statusCode = 500;
        res.send();
    }
}

File: python/controllers/post_unit_to_army_roster_controller.py
from flask import request

from models.post_unit_to_unit_roster_model import post_unit_to_roster_model


def post_unit_roster_controller():
    body = request.get_json()
    if body is None:
        return []
    unit_id = body.get("unit_id", 0)
    if unit_id is 0:
        return []
    return post_unit_to_roster_model(unit_id)
